R/utils.R

```diff
@@ -139,3 +139,34 @@ check_package_available <- function(pkg) {
     stop("Please install suggested package: ", pkg)
   }
 }
+
+# hidden, also in hemibrainr
+add_field_seq <- function(x, entries, field = "bodyid", ...){
```

Comment:
Can you explain what this function is for @alexanderbates?

Reply:
I use it if I wanted to add some information to a neuron, at `neuron$...`. So, if take flywire IDs from the neuron's meta data (as `entries`), and wish to add it at `neuron$flywire.id` (neuronlist is `x`),